dora/core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -985,7 +983,7 @@ public synchronized void removeQuorumServer(NetAddress serverNetAddress) throws
    */
   public synchronized void resetPriorities() throws IOException {
     List<RaftPeer> resetPeers = new ArrayList<>();
-    final int NEUTRAL_PRIORITY = 1;
+    final int NEUTRAL_PRIORITY = 0;
```

Comment:
Did Ratis change its neutral priority from 1 to 0? Or why did we use 1? I don't really remember this part...